build.cake

```diff
@@ -160,137 +105,78 @@ Task("Copy-Files")
     .IsDependentOn("Run-Unit-Tests")
     .Does(() =>
 {
-    CopyFileToDirectory(buildDir + File("Cake.exe"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.Core.dll"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.Core.pdb"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.Core.xml"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.NuGet.dll"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.NuGet.pdb"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.NuGet.xml"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.Common.dll"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.Common.pdb"), binDir);
-    CopyFileToDirectory(buildDir + File("Cake.Common.xml"), binDir);
-    CopyFileToDirectory(buildDir + File("Mono.CSharp.dll"), binDir);
-    CopyFileToDirectory(buildDir + File("Autofac.dll"), binDir);
-    CopyFileToDirectory(buildDir + File("NuGet.Core.dll"), binDir);
-
-    // Copy testing assemblies.
-    var testingDir = Directory("./src/Cake.Testing/bin") + Directory(configuration);
-    CopyFileToDirectory(testingDir + File("Cake.Testing.dll"), binDir);
-    CopyFileToDirectory(testingDir + File("Cake.Testing.pdb"), binDir);
-    CopyFileToDirectory(testingDir + File("Cake.Testing.xml"), binDir);
-
-    CopyFiles(new FilePath[] { "LICENSE", "README.md", "ReleaseNotes.md" }, binDir);
+    CopyFiles(
+        parameters.Paths.Files.ArtifactsSourcePaths,
+        parameters.Paths.Directories.ArtifactsBin
+    );
 });
 
 Task("Zip-Files")
     .IsDependentOn("Copy-Files")
-    .IsDependentOn("Run-GitVersion")
     .Does(() =>
 {
-    var packageFile = File("Cake-bin-v" + semVersion + ".zip");
-    var packagePath = buildResultDir + packageFile;
-
-    var files = GetFiles(binDir.Path.FullPath + "/*")
-      - GetFiles(binDir.Path.FullPath + "/*.Testing.*");
+    var files = GetFiles( parameters.Paths.Directories.ArtifactsBin.FullPath + "/*")
+      - GetFiles(parameters.Paths.Directories.ArtifactsBin.FullPath + "/*.Testing.*");
 
-    Zip(binDir, packagePath, files);
+    Zip(parameters.Paths.Directories.ArtifactsBin, parameters.Paths.Files.ZipArtifactPath, files);
 });
 
 Task("Create-Chocolatey-Packages")
     .IsDependentOn("Copy-Files")
-    .IsDependentOn("Run-GitVersion")
     .IsDependentOn("Package")
-    .WithCriteria(() => isRunningOnWindows)
+    .WithCriteria(() => parameters.IsRunningOnWindows)
```

Comment:
@devlead technically, chocolatey can run on Mac/Linux using mono, but I think that this makes sense.